classes/core-image-pelux.bbclass

```diff
@@ -40,3 +40,4 @@ toolchain_create_sdk_env_script_append() {
 IMAGE_ROOTFS_SIZE ?= "1000000"
 IMAGE_FSTYPES ?= "ext3 sdcard"
 IMAGE_FSTYPES += "ext3.gz"
+IMAGE_FSTYPES_remove = "tar.bz2"
```

Comment:
Is the assignment on line 41 not happening? Seems weird that we would need this _remove line if the assignment is happening. And if it's not happening, it should probably be removed for clarity or changed to a non-conditional assignment to force it.

Also line 42 seems weird if we want ext3.gz it should just be added in the line 41.

Reply:
Isn't the assignment on 41 a strong default, so it may be overridden by, for example, machine configurations? I agree that we should make this clearer though.

Reply:
Yes it can be overridden, which is good. But why are we then explicitly adding ext3.gz and removing tar.bz2 afterwards? What if someone wants tar.bz2? Shouldn't we either set our preferred defaults using ?=, and then not touch if if someone chooses to override it?